client/app/components/form/widgets/SelectWidget.tsx

```diff
@@ -51,12 +51,16 @@ const SelectWidget: React.FC<WidgetProps> = (props) => {
       <InputLabel shrink={false}>{!value && placeholder}</InputLabel>
       <Select
         labelId={id}
+        id={`${id}_select`}
         value={value || ""}
-        id={id}
         disabled={disabled || readonly}
         name={id}
         onChange={handleChange}
         sx={styles}
+        inputProps={{
+          id: id,
+          "aria-label": id,
+        }}
```

Comment:
Fixed an error that revealed itself using `getByLabel`: 
`Found a label with the text of: /Operation Type+/i, however the element associated with this label (<div />) is non-labellable`